pkg/container-disk/container-disk_test.go

```diff
@@ -37,7 +37,10 @@ import (
 )
 
 var _ = Describe("ContainerDisk", func() {
-	tmpDir, _ := ioutil.TempDir("", "containerdisktest")
+	tmpDir, err := ioutil.TempDir("", "containerdisktest")
```

Comment:
Oh that should actually be in a `BeforeEach` block. Would you mind moving it there and use `Expect` instead of `panic`?